packages/@okta/vuepress-site/.vuepress/public/docs/api/postman/identity-sources.json

```diff
@@ -0,0 +1,230 @@
+				},
+				"description": ""
+			},
+			"response": []
+		},
+		{
+			"name": "BulkImport Delete",
+			"request": {
+				"url": "{{url}}/api/v1/identity-sources/{{identitySourceId}}/sessions/{{sessionId}}/bulk-delete",
+				"method": "POST",
+				"header": [
+					{
+						"key": "Accept",
+						"value": "application/json",
+						"description": ""
+					},
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"description": ""
+					},
+					{
+						"key": "Authorization",
+						"value": "SSWS {{apikey}}",
+						"description": ""
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"entityType\": \"USERS\",\n    \"profiles\": [\n      {\n        \"externalId\":\"EXT123456784C1IE\"\n      },\n      {\n        \"externalId\":\"EXT123456784C2IE\"\n      },\n      {\n        \"externalId\":\"EXT123456784C3IE\"\n      }    \n    ]\n}"
+				},
+				"description": ""
+			},
+			"response": []
+		},
+		{
+			"name": "Trigger an import session",
+			"request": {
+				"url": "{{url}}/api/v1/identity-sources/{{identitySourceId}}/sessions/{{sessionId}}/start-import",
+				"method": "POST",
+				"header": [
+					{
+						"key": "Accept",
+						"value": "application/json",
+						"description": ""
+					},
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"description": ""
+					},
+					{
+						"key": "Authorization",
+						"value": "SSWS {{apikey}}",
+						"description": ""
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"description": ""
+			},
+			"response": []
+		},
+		{
+			"name": "Delete an import session",
+			"request": {
+				"url": "{{url}}/api/v1/identity-sources/{{identitySourceId}}/sessions/{{sessionId}}",
+				"method": "DELETE",
+				"header": [
+					{
+						"key": "Accept",
+						"value": "application/json",
+						"description": ""
+					},
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"description": ""
+					},
+					{
+						"key": "Authorization",
+						"value": "SSWS {{apikey}}",
+						"description": ""
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"description": ""
+			},
+			"response": []
+		},
+		{
+			"name": "Get an import session",
+			"request": {
+				"url": "{{url}}/api/v1/identity-sources/{{identitySourceId}}/sessions/{{sessionId}}",
+				"method": "GET",
+				"header": [
+					{
+						"key": "Accept",
+						"value": "application/json",
+						"description": ""
+					},
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"description": ""
+					},
+					{
+						"key": "Authorization",
+						"value": "SSWS {{apikey}}",
+						"description": ""
+					}
+				],
+				"body": {},
+				"description": ""
+			},
+			"response": []
```

Comment:
Why is the response empty?

Reply:
Same answer as above. This is the original JSON collection file you provided.